libs/ui/src/reports/precinct_scanner_ballot_count_report.tsx

```diff
@@ -60,8 +60,8 @@ export function PrecinctScannerBallotCountReport({
   const currentTime = Date.now();
 
   return (
-    <PrintableContainer data-testid="ballot-count-report">
-      <TallyReport>
+    <ThemeProvider theme={TALLY_REPORT_THEME}>
```

Comment:
Is this component ever displayed on-screen? If it is, we might want to preserve the `screenType` param of the pre-existing theme when overriding, since the sizes are specified relative to the screen dimensions/pixel density.

Reply:
Also, if this is a print-only component, I'm wondering if the theme overrides can be avoided if we set the `legacyPrintFontSizePx` prop on the `AppBase` component (e.g. https://github.com/votingworks/vxsuite/blob/main/apps/admin/frontend/src/index.tsx#L27), along with any print-specific `@media` overrides in `global_styles.tsx` (I think there's a pending bug in `global_styles` where we don't specify a color override for `@media print` - I need to fix that)

Reply:
a38607be93d8ca2d17eb8f6fd39b568f365362fa